WhyApp/Protocols/StandardViewControllerProtocol.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Goalable.swift
+//  WhyApp
+//
+//  Created by Joriah Lasater on 6/13/18.
+//  Copyright © 2018 Joriah Lasater. All rights reserved.
+//
+
+import UIKit
+
+protocol StandardViewControllerProtocol: class  {
```

Comment:
Apple put out a nice [API Design Guideline](https://swift.org/documentation/api-design-guidelines/) that details best practices for naming conventions and things like that. One thing that stands out is when naming protocols. 

> - Protocols that describe what something is should read as nouns (e.g. Collection).
> - Protocols that describe a capability should be named using the suffixes able, ible, or ing (e.g. Equatable, ProgressReporting).

That being said, I'd consider renaming this to `StandardViewControllerRepresentable` or something of the like.

Reply:
Okay thanks, finding names for protocols has always been a struggle for me. Is that link the only resource you use when coming up with names for protocols?